python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -579,8 +569,16 @@ def test_groupby_categorical_key():
     dd.assert_eq(expect, got)
 
 
-@xfail_dask_expr("as_index not supported in dask-expr")
-@pytest.mark.parametrize("as_index", [True, False])
+@pytest.mark.parametrize(
+    "as_index",
+    [
+        True,
+        pytest.param(
+            False,
+            marks=xfail_dask_expr("as_index not supported in dask-expr"),
```

Comment:
Is it intended that it will be?

Reply:
I suspect that the only way it will be supported is if we add it upstream. However, I'm not finding any great reasons to add it.